src/howitz/endpoints.py

```diff
@@ -57,18 +63,23 @@
 
 DB_URL = Path('howitz.sqlite3')
 database = UserDB(DB_URL)
-database.initdb()
 
-conf = parse_tcl_config("~/.ritz.tcl")['default']
-session = ritz(
+conf = parse_tcl_config("~/.ritz.tcl")["default"]
+zino_session = ritz(
     conf['Server'],
-    username=conf['User'],
-    password=conf['Secret'],
     timeout=30,
 )
-session.connect()
+event_engine = Zino1EventEngine(zino_session)
+
+
+def initialize_database():
+    database.initdb()
+    app.logger.info('Connected to database %s', database.connection)
 
-event_engine = Zino1EventEngine(session)
+
+def connect_to_zino():
+    zino_session.connect()
```

Comment:
This is a global, not good. zino_session should not be directly visible to howitz after event_engine is initialized. This can stay until SessionAdapter is in.